### ISSUE TITLE
Error on Firefox

### ISSUE THREAD

**Issue by tshino** [tshino/compare#159]
I found some PNG images that cannot load in this app on Firefox.

An example could be created with this code:
```py
import cv2
import numpy as np
img = np.random.randint(np.ones((4096,4096,4))*256)
cv2.imwrite('random_rgba_cv2.png', img)
```

**Comment by tshino** [comment]
Fixed.

**Comment by tshino** [comment]
I found the above fix of this issue caused another problem #160.
